packages/dataviews/src/field-types/text.tsx

```diff
@@ -67,6 +67,8 @@ function Edit< Item >( {
 				value={ value }
 				options={ elements }
 				onChange={ onChangeControl }
+				__next40pxDefaultSize
+				__nextHasNoMarginBottom
```

Comment:
No problem if this PR merges first, but FYI these props are being added in bulk to all in-app SelectControls in #64283.